app/controllers/news_controller.rb

```diff
@@ -6,9 +6,28 @@ def index
     end
 
     def search
-	    @keyword = params[:q]
+	@keyword = params[:q]
         @items = News.order(id: :desc)
                      .where("title ILIKE ?", "%#{@keyword}%").limit(20)
         render "search", layout: "application"
     end
+
+ 
+    def searchLikeUp
```

Comment:
Uuuh, eu estou confuso.

Isso daqui era para ser um contador de notícia acessada ou para ser um esquema de up/downvote? Porque acho que são duas coisas diferentes.

Reply:
Então, eu montei esse serviço de upvote/downvote para testar mesmo como funciona esse mecanismo. Eu vou fazer um processo reverso agora, que transforma o que eu tenho em um sistema de acessos só. Isto é, tirar o downvote e apenas realizar a contagem de "clicks" que os usuários logados no sistema dão no botão "Ler notícia". Em seguida, vai ter uma página que faz somente isso.